src/Phinx/Console/Command/Create.php

```diff
@@ -201,7 +201,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         // Get the alternative template and static class options from the command line, but only allow one of them.
+        /** @var string|null */
```

Comment:
the variable is missing here

Reply:
Added.

The variable isn't strictly necessary as the IDE / phpstan will just apply whatever is there then to the variable in the next line, though having the variable is clearer.